getting_started/scripting/gdscript/static_typing.rst

```diff
@@ -0,0 +1,407 @@
+.. _doc_gdscript_static_typing:
+
+Static Typing in GDScript
+=========================
+
+In this guide, you will learn:
+
+-  **How to use types in GDScript**
+-  That **static types can help you avoid bugs**
+
+Where and how you use this new language feature is entirely up to you:
+you can use it only in some sensitive GDScript files, use it everywhere,
+or write code like you always did!
+
+Static types can be used on variables, constants, functions, parameters,
+and return types.
+
+.. note::
+
+    Typed GDScript is available since Godot 3.1.
```

Comment:
Just a thought, but since this is going in master, do we need this here? You see the 3.0 branch of the docs by default, not the master branch. By the time this gets seen in the default branch, 3.1 will have already been released.

Reply:
Hence "since". It's worth noting for people who have an ongoing production with Godot 3.0 and may take 6 months to 1 year before they upgrade - they're still likely to land on the stable docs, e.g. searching on Google. Same for Godot 2.x users

Reply:
Okay, that's fair. I didn't think of that.